contracts/extensions/contracts/AllocatorsAllowlistExtension.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
```

Comment:
Let's keep the `^` since we changed it for all contracts on the PR that we bumped the sol version

Reply:
Pushed that change here and in #654.
Anyway, we need to merge #652 before the extensions PRs.